packages/now-node/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "strict": true,
     "esModuleInterop": true,
     "lib": ["esnext"],
-    "target": "esnext",
+    "target": "es2018",
```

Comment:
Needed to specify this target for compatibility with Node 10+ in order to use the `foo?.` syntax.